core/src/main/java/io/opentelemetry/android/features/diskbuffering/DiskBufferingConfig.kt

```diff
@@ -26,6 +27,7 @@ data class DiskBufferingConfig
         val maxFileAgeForReadMillis: Long = TimeUnit.HOURS.toMillis(DEFAULT_MAX_FILE_AGE_FOR_READ_MS),
         val maxCacheFileSize: Int = MAX_CACHE_FILE_SIZE,
         val debugEnabled: Boolean = false,
+        val exportIntervalInSeconds: Long = DEFAULT_EXPORT_INTERVAL_IN_SECONDS,
```

Comment:
should we use `Duration` instead of `Long`?
eg https://github.com/open-telemetry/opentelemetry-android/blob/c2ddbe09784e1a5e24d16ac29b876e245153e138/session/src/main/kotlin/io/opentelemetry/android/session/SessionConfig.kt#L13-L14